osgp/protocol-adapter-iec60870/osgp-protocol-simulator-iec60870/pom.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Smart Society Services B.V. Licensed under the Apache License, 
+  Version 2.0 (the "License"); you may not use this file except in compliance with 
+  the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 -->
+<project
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.1.3.RELEASE</version>
+    <relativePath /> <!-- lookup parent from repository -->
```

Comment:
This file was automatically added by Spring Initializr, as such this line was also included.
Removing this line results in the following warning when building using maven:

> 'parent.relativePath' of POM org.opensmartgridplatform:osgp-protocol-simulator-iec60870:4.35.0-SNAPSHOT points at org.opensmartgridplatform:protocol-adapter-iec60870 instead of org.springframework.boot:spring-boot-starter-parent, please verify your project structure @ line 12, column 11